Cargo.toml

```diff
@@ -11,7 +11,7 @@ name = "lib"
 path = "src/lib.rs"
 
 [features]
-default = ["base-import", "btrfs", "grub", "import", "pacman"]
```

Comment:
We should find a way to avoid changing this back and forth between distros. A way to dynamically switch between these "very base" critical packages for that specific distro.

Reply:
In this code, the answer is no.

Reply:
I need to explain what I did and why in a long message 😅.

Reply:
Well, I'm noticing package managers are being switched every now and then. Even if not possible this way, I think a heuristic way should be put in place to kind of dynamically do this.

Reply:
I don't see this as a major issue.

Reply:
By the way, the code is ready for test. I wish if I could make the installer work properly 🥲.

Reply:
I'm planning to get it fixed as soon as possible.

Reply:
https://github.com/ochibani/ash-installer